packages/templates/src/templates/package.json.hbs

```diff
@@ -19,8 +19,8 @@
     {{#if_eq contract_language "ask"}}
     "ask-lang": "^0.4.0-rc3",
     {{/if_eq}}
-    "@supercolony/typechain-compiler": "^0.5.1-beta.1",
-    "@supercolony/typechain-types": "^0.0.16",
+    "@727-ventures/typechain-compiler": "^0.5.13",
+    "@727-ventures/typechain-types": "^0.0.22",
     "typescript": "^4.9.3",
     "patch-package": "^6.4.7"
```

Comment:
this can also go since we're not using the patch.
Also, can you remove the `^` in the template too, so we keep it consistent?